evalml/automl/automl_search.py

```diff
@@ -256,6 +259,8 @@ def __init__(self,
 
         self._validate_problem_type()
         self.problem_configuration = self._validate_problem_configuration(problem_configuration)
+        self._train_best_pipe = train_best_pipeline
+        self._best_pipe = None
```

Comment:
This is total style nit-pick but if you say `pipeline` instead of `pipe` for these variable names it'll potentially make it easier to grep through the code 😅